package.json

```diff
@@ -79,9 +79,17 @@
   "pre-commit": "lint:staged",
   "dependencies": {
     "@babel/helper-regex": "^7.10.5",
+    "@emotion/core": "^11.0.0",
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
     "@formatjs/intl-relativetimeformat": "^9.1.6",
+    "@mui/icons-material": "^5.11.9",
+    "@mui/material": "^5.11.10",
+    "@mui/styled-engine": "^5.11.9",
+    "@mui/styled-engine-sc": "^5.11.9",
```

Comment:
Do we need these ? These are for styled components. Please check

Reply:
removed